shakenfist/daemons/resources.py

```diff
@@ -48,12 +48,12 @@ def _get_stats():
     except Exception:
         pass
 
-    # Memory info
+    # Memory info - libvirt returns memory in KiB
     memory_status = conn.getMemoryStats(
         libvirt.VIR_NODE_MEMORY_STATS_ALL_CELLS)
     retval.update({
-        'memory_max': memory_status['total'],
-        'memory_available': memory_status['free'],
+        'memory_max': memory_status['total'] // 1024,
+        'memory_available': memory_status['free'] // 1024,
```

Comment:
Oh good catch. This will make the dashboards nicer. I wonder how it affects the scheduler (if at all)?